<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>840</width>
    <height>450</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="minimumSize">
   <size>
    <width>840</width>
    <height>450</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>840</width>
    <height>450</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QPushButton" name="pushButton_del">
    <property name="geometry">
     <rect>
      <x>15</x>
      <y>150</y>
      <width>55</width>
      <height>55</height>
     </rect>
    </property>
    <property name="text">
     <string>&lt;&lt;</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_div">
    <property name="geometry">
     <rect>
      <x>190</x>
      <y>205</y>
      <width>55</width>
      <height>55</height>
     </rect>
    </property>
    <property name="text">
     <string>/</string>
    </property>
    <attribute name="buttonGroup">
     <string notr="true">GSingleSymbolButton</string>
    </attribute>
   </widget>
   <widget class="QPushButton" name="pushButton_dot">
    <property name="geometry">
     <rect>
      <x>125</x>
      <y>340</y>
      <width>55</width>
      <height>55</height>
     </rect>
    </property>
    <property name="sizePolicy">
     <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
      <horstretch>0</horstretch>
      <verstretch>0</verstretch>
     </sizepolicy>
    </property>
    <property name="text">
     <string>.</string>
    </property>
    <attribute name="buttonGroup">
     <string notr="true">GSingleSymbolButton</string>
    </attribute>
   </widget>
   <widget class="QPushButton" name="pushButton_L_bracket">
    <property name="geometry">
     <rect>
      <x>310</x>
      <y>250</y>
      <width>55</width>
      <height>55</height>
     </rect>
    </property>
    <property name="text">
     <string>)</string>
    </property>
    <attribute name="buttonGroup">
     <string notr="true">GSingleSymbolButton</string>
    </attribute>
   </widget>
   <widget class="QPushButton" name="pushButton_3">
    <property name="geometry">
     <rect>
      <x>125</x>
      <y>295</y>
      <width>55</width>
      <height>55</height>
     </rect>
    </property>
    <property name="sizePolicy">
     <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
      <horstretch>0</horstretch>
      <verstretch>0</verstretch>
     </sizepolicy>
    </property>
    <property name="text">
     <string>3</string>
    </property>
    <attribute name="buttonGroup">
     <string notr="true">GSingleSymbolButton</string>
    </attribute>
   </widget>
   <widget class="QPushButton" name="pushButton_sqrt">
    <property name="geometry">
     <rect>
      <x>310</x>
      <y>340</y>
      <width>55</width>
      <height>55</height>
     </rect>
    </property>
    <property name="text">
     <string>sqrt</string>
    </property>
    <attribute name="buttonGroup">
     <string notr="true">GFuncButton</string>
    </attribute>
   </widget>
   <widget class="QPushButton" name="pushButton_1">
    <property name="geometry">
     <rect>
      <x>15</x>
      <y>295</y>
      <width>55</width>
      <height>55</height>
     </rect>
    </property>
    <property name="text">
     <string>1</string>
    </property>
    <attribute name="buttonGroup">
     <string notr="true">GSingleSymbolButton</string>
    </attribute>
   </widget>
   <widget class="QLineEdit" name="CalcLine">
    <property name="geometry">
     <rect>
      <x>16</x>
      <y>15</y>
      <width>460</width>
      <height>65</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>24</pointsize>
     </font>
    </property>
    <property name="maxLength">
     <number>255</number>
    </property>
    <property name="frame">
     <bool>false</bool>
    </property>
    <property name="alignment">
     <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_8">
    <property name="geometry">
     <rect>
      <x>70</x>
      <y>205</y>
      <width>55</width>
      <height>55</height>
     </rect>
    </property>
    <property name="text">
     <string>8</string>
    </property>
    <attribute name="buttonGroup">
     <string notr="true">GSingleSymbolButton</string>
    </attribute>
   </widget>
   <widget class="QPushButton" name="pushButton_sin">
    <property name="geometry">
     <rect>
      <x>365</x>
      <y>205</y>
      <width>55</width>
      <height>55</height>
     </rect>
    </property>
    <property name="text">
     <string>sin</string>
    </property>
    <attribute name="buttonGroup">
     <string notr="true">GFuncButton</string>
    </attribute>
   </widget>
   <widget class="QPushButton" name="pushButton_ln">
    <property name="geometry">
     <rect>
      <x>420</x>
      <y>340</y>
      <width>55</width>
      <height>55</height>
     </rect>
    </property>
    <property name="text">
     <string>ln</string>
    </property>
    <attribute name="buttonGroup">
     <string notr="true">GFuncButton</string>
    </attribute>
   </widget>
   <widget class="QPushButton" name="pushButton_0">
    <property name="geometry">
     <rect>
      <x>15</x>
      <y>340</y>
      <width>110</width>
      <height>55</height>
     </rect>
    </property>
    <property name="text">
     <string>0</string>
    </property>
    <attribute name="buttonGroup">
     <string notr="true">GSingleSymbolButton</string>
    </attribute>
   </widget>
   <widget class="QPushButton" name="pushButton_plus">
    <property name="geometry">
     <rect>
      <x>190</x>
      <y>340</y>
      <width>55</width>
      <height>55</height>
     </rect>
    </property>
    <property name="text">
     <string>+</string>
    </property>
    <attribute name="buttonGroup">
     <string notr="true">GSingleSymbolButton</string>
    </attribute>
   </widget>
   <widget class="QPushButton" name="pushButton_x">
    <property name="geometry">
     <rect>
      <x>124</x>
      <y>150</y>
      <width>55</width>
      <height>55</height>
     </rect>
    </property>
    <property name="text">
     <string>x</string>
    </property>
    <attribute name="buttonGroup">
     <string notr="true">GSingleSymbolButton</string>
    </attribute>
   </widget>
   <widget class="QPushButton" name="pushButton_acos">
    <property name="geometry">
     <rect>
      <x>420</x>
      <y>250</y>
      <width>55</width>
      <height>55</height>
     </rect>
    </property>
    <property name="text">
     <string>acos</string>
    </property>
    <attribute name="buttonGroup">
     <string notr="true">GFuncButton</string>
    </attribute>
   </widget>
   <widget class="QPushButton" name="pushButton_7">
    <property name="geometry">
     <rect>
      <x>15</x>
      <y>205</y>
      <width>55</width>
      <height>55</height>
     </rect>
    </property>
    <property name="text">
     <string>7</string>
    </property>
    <attribute name="buttonGroup">
     <string notr="true">GSingleSymbolButton</string>
    </attribute>
   </widget>
   <widget class="QLineEdit" name="Var_line">
    <property name="geometry">
     <rect>
      <x>190</x>
      <y>155</y>
      <width>110</width>
      <height>45</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>24</pointsize>
     </font>
    </property>
    <property name="maxLength">
     <number>255</number>
    </property>
    <property name="frame">
     <bool>false</bool>
    </property>
    <property name="alignment">
     <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
    </property>
    <property name="readOnly">
     <bool>false</bool>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_pow">
    <property name="geometry">
     <rect>
      <x>310</x>
      <y>295</y>
      <width>55</width>
      <height>55</height>
     </rect>
    </property>
    <property name="text">
     <string>^</string>
    </property>
    <attribute name="buttonGroup">
     <string notr="true">GSingleSymbolButton</string>
    </attribute>
   </widget>
   <widget class="QPushButton" name="pushButton_cos">
    <property name="geometry">
     <rect>
      <x>365</x>
      <y>250</y>
      <width>55</width>
      <height>55</height>
     </rect>
    </property>
    <property name="text">
     <string>cos</string>
    </property>
    <attribute name="buttonGroup">
     <string notr="true">GFuncButton</string>
    </attribute>
   </widget>
   <widget class="QPushButton" name="pushButton_minus">
    <property name="geometry">
     <rect>
      <x>190</x>
      <y>295</y>
      <width>55</width>
      <height>55</height>
     </rect>
    </property>
    <property name="text">
     <string>-</string>
    </property>
    <attribute name="buttonGroup">
     <string notr="true">GSingleSymbolButton</string>
    </attribute>
   </widget>
   <widget class="QPushButton" name="pushButton_mul">
    <property name="geometry">
     <rect>
      <x>190</x>
      <y>250</y>
      <width>55</width>
      <height>55</height>
     </rect>
    </property>
    <property name="text">
     <string>*</string>
    </property>
    <attribute name="buttonGroup">
     <string notr="true">GSingleSymbolButton</string>
    </attribute>
   </widget>
   <widget class="QPushButton" name="pushButton_asin">
    <property name="geometry">
     <rect>
      <x>420</x>
      <y>205</y>
      <width>55</width>
      <height>55</height>
     </rect>
    </property>
    <property name="text">
     <string>asin</string>
    </property>
    <attribute name="buttonGroup">
     <string notr="true">GFuncButton</string>
    </attribute>
   </widget>
   <widget class="QPushButton" name="pushButton_6">
    <property name="geometry">
     <rect>
      <x>125</x>
      <y>250</y>
      <width>55</width>
      <height>55</height>
     </rect>
    </property>
    <property name="sizePolicy">
     <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
      <horstretch>0</horstretch>
      <verstretch>0</verstretch>
     </sizepolicy>
    </property>
    <property name="text">
     <string>6</string>
    </property>
    <attribute name="buttonGroup">
     <string notr="true">GSingleSymbolButton</string>
    </attribute>
   </widget>
   <widget class="QPushButton" name="pushButton_credit">
    <property name="geometry">
     <rect>
      <x>250</x>
      <y>205</y>
      <width>55</width>
      <height>55</height>
     </rect>
    </property>
    <property name="text">
     <string>credit</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_eq">
    <property name="geometry">
     <rect>
      <x>250</x>
      <y>340</y>
      <width>55</width>
      <height>55</height>
     </rect>
    </property>
    <property name="text">
     <string>=</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_5">
    <property name="geometry">
     <rect>
      <x>70</x>
      <y>250</y>
      <width>55</width>
      <height>55</height>
     </rect>
    </property>
    <property name="text">
     <string>5</string>
    </property>
    <attribute name="buttonGroup">
     <string notr="true">GSingleSymbolButton</string>
    </attribute>
   </widget>
   <widget class="QPushButton" name="pushButton_4">
    <property name="geometry">
     <rect>
      <x>15</x>
      <y>250</y>
      <width>55</width>
      <height>55</height>
     </rect>
    </property>
    <property name="text">
     <string>4</string>
    </property>
    <attribute name="buttonGroup">
     <string notr="true">GSingleSymbolButton</string>
    </attribute>
   </widget>
   <widget class="QLineEdit" name="ResLine">
    <property name="geometry">
     <rect>
      <x>16</x>
      <y>80</y>
      <width>460</width>
      <height>65</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>24</pointsize>
     </font>
    </property>
    <property name="maxLength">
     <number>255</number>
    </property>
    <property name="frame">
     <bool>false</bool>
    </property>
    <property name="alignment">
     <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
    </property>
    <property name="readOnly">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_log">
    <property name="geometry">
     <rect>
      <x>365</x>
      <y>340</y>
      <width>55</width>
      <height>55</height>
     </rect>
    </property>
    <property name="text">
     <string>log</string>
    </property>
    <attribute name="buttonGroup">
     <string notr="true">GFuncButton</string>
    </attribute>
   </widget>
   <widget class="QPushButton" name="pushButton_AC">
    <property name="geometry">
     <rect>
      <x>70</x>
      <y>150</y>
      <width>55</width>
      <height>55</height>
     </rect>
    </property>
    <property name="text">
     <string>C</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_L_bracket_2">
    <property name="geometry">
     <rect>
      <x>310</x>
      <y>205</y>
      <width>55</width>
      <height>55</height>
     </rect>
    </property>
    <property name="text">
     <string>(</string>
    </property>
    <attribute name="buttonGroup">
     <string notr="true">GSingleSymbolButton</string>
    </attribute>
   </widget>
   <widget class="QPushButton" name="pushButton_atan">
    <property name="geometry">
     <rect>
      <x>420</x>
      <y>295</y>
      <width>55</width>
      <height>55</height>
     </rect>
    </property>
    <property name="text">
     <string>atan</string>
    </property>
    <attribute name="buttonGroup">
     <string notr="true">GFuncButton</string>
    </attribute>
   </widget>
   <widget class="QPushButton" name="pushButton_9">
    <property name="geometry">
     <rect>
      <x>125</x>
      <y>205</y>
      <width>55</width>
      <height>55</height>
     </rect>
    </property>
    <property name="sizePolicy">
     <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
      <horstretch>0</horstretch>
      <verstretch>0</verstretch>
     </sizepolicy>
    </property>
    <property name="text">
     <string>9</string>
    </property>
    <attribute name="buttonGroup">
     <string notr="true">GSingleSymbolButton</string>
    </attribute>
   </widget>
   <widget class="QPushButton" name="pushButton_tan">
    <property name="geometry">
     <rect>
      <x>365</x>
      <y>295</y>
      <width>55</width>
      <height>55</height>
     </rect>
    </property>
    <property name="text">
     <string>tan</string>
    </property>
    <attribute name="buttonGroup">
     <string notr="true">GFuncButton</string>
    </attribute>
   </widget>
   <widget class="QPushButton" name="pushButton_2">
    <property name="geometry">
     <rect>
      <x>70</x>
      <y>295</y>
      <width>55</width>
      <height>55</height>
     </rect>
    </property>
    <property name="text">
     <string>2</string>
    </property>
    <attribute name="buttonGroup">
     <string notr="true">GSingleSymbolButton</string>
    </attribute>
   </widget>
   <widget class="QLabel" name="Max_x">
    <property name="geometry">
     <rect>
      <x>490</x>
      <y>336</y>
      <width>58</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Max X</string>
    </property>
   </widget>
   <widget class="QLabel" name="Min_x">
    <property name="geometry">
     <rect>
      <x>490</x>
      <y>370</y>
      <width>58</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Min X</string>
    </property>
   </widget>
   <widget class="QLabel" name="Max_y">
    <property name="geometry">
     <rect>
      <x>670</x>
      <y>336</y>
      <width>58</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Max Y</string>
    </property>
   </widget>
   <widget class="QLabel" name="Min_x_2">
    <property name="geometry">
     <rect>
      <x>670</x>
      <y>370</y>
      <width>58</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Min Y</string>
    </property>
   </widget>
   <widget class="QDoubleSpinBox" name="DSB_max_x">
    <property name="geometry">
     <rect>
      <x>550</x>
      <y>332</y>
      <width>101</width>
      <height>22</height>
     </rect>
    </property>
    <property name="decimals">
     <number>1</number>
    </property>
    <property name="minimum">
     <double>-100000.000000000000000</double>
    </property>
    <property name="maximum">
     <double>100000.000000000000000</double>
    </property>
    <property name="singleStep">
     <double>1.000000000000000</double>
    </property>
    <property name="value">
     <double>5.000000000000000</double>
    </property>
   </widget>
   <widget class="QDoubleSpinBox" name="DSB_min_x">
    <property name="geometry">
     <rect>
      <x>550</x>
      <y>367</y>
      <width>101</width>
      <height>22</height>
     </rect>
    </property>
    <property name="decimals">
     <number>1</number>
    </property>
    <property name="minimum">
     <double>-100000.000000000000000</double>
    </property>
    <property name="maximum">
     <double>100000.000000000000000</double>
    </property>
    <property name="singleStep">
     <double>1.000000000000000</double>
    </property>
    <property name="value">
     <double>-5.000000000000000</double>
    </property>
   </widget>
   <widget class="QDoubleSpinBox" name="DSB_min_y">
    <property name="geometry">
     <rect>
      <x>730</x>
      <y>367</y>
      <width>101</width>
      <height>22</height>
     </rect>
    </property>
    <property name="decimals">
     <number>1</number>
    </property>
    <property name="minimum">
     <double>-100000.000000000000000</double>
    </property>
    <property name="maximum">
     <double>100000.000000000000000</double>
    </property>
    <property name="singleStep">
     <double>1.000000000000000</double>
    </property>
    <property name="value">
     <double>-5.000000000000000</double>
    </property>
   </widget>
   <widget class="QDoubleSpinBox" name="DSB_max_y">
    <property name="geometry">
     <rect>
      <x>730</x>
      <y>332</y>
      <width>101</width>
      <height>22</height>
     </rect>
    </property>
    <property name="decimals">
     <number>1</number>
    </property>
    <property name="minimum">
     <double>-100000.000000000000000</double>
    </property>
    <property name="maximum">
     <double>100000.000000000000000</double>
    </property>
    <property name="singleStep">
     <double>1.000000000000000</double>
    </property>
    <property name="value">
     <double>5.000000000000000</double>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_since_not">
    <property name="geometry">
     <rect>
      <x>250</x>
      <y>250</y>
      <width>55</width>
      <height>55</height>
     </rect>
    </property>
    <property name="text">
     <string>E</string>
    </property>
    <attribute name="buttonGroup">
     <string notr="true">GSingleSymbolButton</string>
    </attribute>
   </widget>
   <widget class="QLabel" name="Min_x_3">
    <property name="geometry">
     <rect>
      <x>310</x>
      <y>180</y>
      <width>91</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Points for plot:</string>
    </property>
   </widget>
   <widget class="QSpinBox" name="DotsSpinBox">
    <property name="geometry">
     <rect>
      <x>400</x>
      <y>177</y>
      <width>71</width>
      <height>22</height>
     </rect>
    </property>
    <property name="minimum">
     <number>2</number>
    </property>
    <property name="maximum">
     <number>1000000</number>
    </property>
    <property name="value">
     <number>500</number>
    </property>
   </widget>
   <widget class="QCustomPlot" name="graph_widget" native="true">
    <property name="geometry">
     <rect>
      <x>500</x>
      <y>15</y>
      <width>310</width>
      <height>310</height>
     </rect>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_eq_2">
    <property name="geometry">
     <rect>
      <x>250</x>
      <y>295</y>
      <width>55</width>
      <height>55</height>
     </rect>
    </property>
    <property name="text">
     <string>mod</string>
    </property>
    <attribute name="buttonGroup">
     <string notr="true">GSingleSymbolButton</string>
    </attribute>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>840</width>
     <height>24</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <customwidgets>
  <customwidget>
   <class>QCustomPlot</class>
   <extends>QWidget</extends>
   <header location="global">qcustomplot.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
 <buttongroups>
  <buttongroup name="GSingleSymbolButton"/>
  <buttongroup name="GFuncButton"/>
 </buttongroups>
</ui>
